gettingstarted/settings.py

```diff
@@ -105,15 +127,41 @@
 
 USE_I18N = True
 
-USE_L10N = True
-
 USE_TZ = True
 
 
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.0/howto/static-files/
 
-STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
 STATIC_URL = "/static/"
 
-django_heroku.settings(locals())
+# Ensure STATIC_ROOT exists.
+
+os.makedirs(STATIC_ROOT, exist_ok=True)
+
```

Comment:
```suggestion
```

This isn't needed since `collectstatic` automatically makes the directory. This can be confirmed by making sure the directory doesn't exist locally, then running `./manage.py collectstatic` and seeing it succeeds,  eg:

```
(.venv) gettingstarted $ ./manage.py collectstatic

128 static files copied to '/Users/emorley/src/django-test/gettingstarted/staticfiles', 378 post-processed.
```